[UPDATE BONUS GUIDE] Sparrow Terminal to version 1.7.3 (#1258)
raspibolt/raspibolt: guide/bonus/bitcoin/sparrow-terminal.md

### Install Sparrow Server

> * Look up the [latest Sparrow Wallet release version](https://github.com/sparrowwallet/sparrow/releases), change "$VERSION" value in the prepared links to that number to download Sparrow 

Reviewer: You must declare the environment variable before use it

Author: Thanks, commited

  ```sh

>   $ cd /tmp

Reviewer: Add:

```sh
$ VERSION=1.7.3
```

>   $ cd /tmp

Reviewer 2: ```suggestion
  $ VERSION="1.7.3"
  $ cd /tmp
```

Reviewer 2: This will resolve the issue Twofaktor is referring to.

Author: Thanks! will commit

### Install Sparrow Server

> * Look up the [latest Sparrow Wallet release version](https://github.com/sparrowwallet/sparrow/releases), change "$VERSION" value in the prepared links to that number to download Sparrow 

Reviewer: ```suggestion
* Download Sparrow Server and signatures into "/tmp" directory, which is cleared on the reboot. If you want to update or install the latest version, look up the [latest Sparrow Wallet release version](https://github.com/sparrowwallet/sparrow/releases), and change the "VERSION="x.x.x" value line, to that number to download the latest Sparrow server binary.
```

* Verify the release
  
  ```sh
  $ gpg --verify sparrow-$VERSION-manifest.txt.asc
  ```
  ```

>   > gpg: assuming signed data in 'sparrow-$VERSION-manifest.txt'

Reviewer 2: ```suggestion
  > gpg: assuming signed data in 'sparrow-1.7.3-manifest.txt'
```
This is one of the couple of places you need to leave it as the actual version number. Since it is the output from a command rather than the command itself.

  ```
  
  ```sh
  $ sha256sum --check sparrow-$VERSION-manifest.txt --ignore-missing

>   > sparrow-server-$VERSION-aarch64.tar.gz: OK

Reviewer 2: ```suggestion
  > sparrow-server-1.7.3-aarch64.tar.gz: OK
```
This is the other place you need to leave it as the actual version number. Since it is the output from a command rather than the command itself.
